resources/data/layers/tree.json

```diff
@@ -114,14 +114,14 @@
             "leaf": true,
             "text": "OpenStreetMap",
             "qtip": "An OpenStreetMap based background layer",
-            "iconCls": "map"
+            "cls": "cpsi-tree-node-baselayer"
```

Comment:
Both cls and iconCls can be kept on the nodes - the CSS sets the checkbox style, whereas the iconClass adds an additional icon

Reply:
I know, but we don't have any meaningful icons (besides the default ones) in the demo application. Setting `"iconCls": "map"` would end in no visible icon.

Reply:
ok sorry was looking at my other application! Feel free to merge